backend/pkg/log/log.go

```diff
@@ -289,3 +291,46 @@ func FromContext(ctx context.Context) *Logger {
 func WithContext(ctx context.Context, log *Logger) context.Context {
 	return context.WithValue(ctx, loggerContextKey, log)
 }
+
+type RecoveryOption struct {
+	channel chan error
+	err     error
+}
+
+func NewRecoveryOption() *RecoveryOption {
+	return &RecoveryOption{}
+}
+
+func (o *RecoveryOption) WithChannel(channel chan error) *RecoveryOption {
+	o.channel = channel
+	return o
+}
+
+func (o *RecoveryOption) WithError(err error) *RecoveryOption {
+	o.err = err
+	return o
+}
+
+func (l *Logger) SimpleRecovery(opts ...*RecoveryOption) {
+	r := recover()
+	if r == nil {
+		return
+	}
+	var err error
+	var opt *RecoveryOption
+	if len(opts) > 0 {
+		opt = opts[0]
+	}
+
+	if opt == nil || opt.err == nil {
+		err = ErrPanic
+	} else {
+		err = errors.Wrap(opt.err, ErrPanic.Error())
+	}
+
+	if opt != nil && opt.channel != nil {
+		opt.channel <- opt.err
+	}
+
+	l.WithField("panic", r).Error(err)
```

Comment:
Arguably the most important information after a panic is the stack trace. Could you add another field `trace` with the stack trace starting from the location of the panic?
Hint: checkout the access log middleware ;)